.jules/runs/run-bolt-core-001/pr_body.md

```
@@ -0,0 +1,56 @@
+## 💡 Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)